modules/integration/tests-common/web-ui-pages/src/main/java/org/wso2/iot/integration/ui/pages/error/IOTErrorPage.java

```diff
@@ -0,0 +1,31 @@
+package org.wso2.iot.integration.ui.pages.error;
```

Comment:
Missing license header is added in commit https://github.com/wso2/product-iots/commit/399136daae1b5183134e5b3da6c5fef9fa633f47#diff-72a5d3c4976bfae9d110af65ad079da7